src/WorkOS.net/Services/DirectorySync/Enums/DirectoryState.cs

```diff
@@ -10,11 +10,11 @@
     [JsonConverter(typeof(StringEnumConverter))]
     public enum DirectoryState
     {
-        [EnumMember(Value = "linked")]
-        Linked,
+        [EnumMember(Value = "active")]
```

Comment:
The API is still going to send back the values `linked` and `unlinked`, so we need to keep the `EnumMember` value the same, but it will map to the new values in the .NET type system.

```suggestion
        [EnumMember(Value = "linked")]
```